### PR TITLE
boards: apollo3: README: Update LoRa examples

### PR DESCRIPTION
**Pull request by alistair23**
### Pull Request Overview

Update the README to include some more details on running the LoRa examples

### Documentation Updated

- [X] Updated the relevant files in `/docs`, or no updates are required.

### Formatting

- [X] Ran `make prepush`.

### CODE REVIEW

**Review by bradjc — DISMISSED** [review]

File: boards/apollo3/lora_things_plus/README.md
Comment (alevy) [inline boards/apollo3/lora_things_plus/README.md:120]: It looks like the PR might be merged post-haste. Does it make sense to wait on that, update the README, then merge?

Comment (alistair23) [reply boards/apollo3/lora_things_plus/README.md:120]: I'm hoping it will be, but it's hard to know for sure. I'm happy to just update the README once it is merged

**Review by alevy — APPROVED** [review]

File: boards/apollo3/lora_things_plus/README.md
Comment (alistair23) [reply boards/apollo3/lora_things_plus/README.md]: Updated with the upstream status!

**Review by bradjc — APPROVED** [review]

**Pull request merged**
